lib/src/main/java/org/team100/lib/util/CameraAngles.java

```diff
@@ -0,0 +1,197 @@
+package org.team100.lib.util;
+
+import org.team100.lib.config.Identity;
+import org.team100.lib.localization.NotePosition24ArrayListener;
+import org.team100.lib.motion.drivetrain.SwerveDriveSubsystem;
+import org.team100.lib.telemetry.Telemetry;
+import org.team100.lib.telemetry.Telemetry.Level;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Translation2d;
+
+public class CameraAngles {
```

Comment:
it would be good for this class to *just* do the math part, and not know about things like the listener or the swerve drive.  put that stuff somewhere else.